content/en/docs/developer/mappers/_index.md

```diff
@@ -10,7 +10,24 @@ toc: true
 type: docs
 ---
 
+## Overview
+Mapper is an application that is used to connect and control devices. Following are the responsibilities of mapper:
 
-- [bluetooth](./bluetooth)
+- Scan and connect to the device.
+- Report the actual state of twin-attributes of device.
+- Map the expected state of device-twin to actual state of device-twin.
+- Collect telemetry data from device.
+- Convert readings from device to format accepted by KubeEdge.
+- Schedule actions on the device.
```

Comment:
"Schedule actions on the device." What function does this item refer to?

Reply:
The function is referring to the [scheduler](https://kubeedge.io/en/docs/developer/mappers/bluetooth/#scheduler) module.

Reply:
Thank you. lgtm :)